src/api/middleware/validators/offer.js

```diff
@@ -11,13 +11,44 @@ const OfferConstants = require("../../../models/constants/Offer");
 const Company = require("../../../models/Company");
 const { isObjectId } = require("../validators/validatorUtils");
 const Offer = require("../../../models/Offer");
+const { validatePublishEndDateLimit } = require("../../../models/Offer");
 const { ErrorTypes } = require("../errorHandler");
 const HTTPStatus = require("http-status-codes");
 const {
     HOUR_IN_MS,
     OFFER_EDIT_GRACE_PERIOD_HOURS,
+    MONTH_IN_MS,
+    OFFER_MAX_LIFETIME_MONTHS
 } = require("../../../models/constants/TimeConstants");
 
+
+const publishEndDateBeforePublishDate = (publishEndDateCandidate, { req }) => {
```

Comment:
Change the name to `publishEndDateAfterPublishDate`, since that's what is really being enforced